pkg/operator/k8sutil/network.go

```diff
@@ -21,12 +21,31 @@ import (
 	"fmt"
 	"strings"
 
+	netapi "github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/apis/k8s.cni.cncf.io/v1"
 	rookv1 "github.com/rook/rook/pkg/apis/rook.io/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// NetworkAttachmentConfig represents the configuration of the NetworkAttachmentDefinitions object
+type NetworkAttachmentConfig struct {
+	CniVersion string `json:"cniVersion"`
+	Type       string `json:"type"`
+	Master     string `json:"master"`
+	Mode       string `json:"mode"`
+	Ipam       struct {
+		Type       string `json:"type"`
+		Subnet     string `json:"subnet"`
+		RangeStart string `json:"rangeStart"`
+		RangeEnd   string `json:"rangeEnd"`
+		Routes     []struct {
+			Dst string `json:"dst"`
+		} `json:"routes"`
+		Gateway string `json:"gateway"`
+	} `json:"ipam"`
+}
+
 // parseMultusSelector will parse short and JSON form of individual multus
-// network attachment selection annotation. Valid JSON will be unmarshaled and
+// network attachment selection annotation. Valid JSON will be unmarshal and
```

Comment:
nit: this looked more correct before the change

Reply:
I guess  `unmarshalled` is correct now.